public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -37,7 +37,7 @@ export interface DashboardSceneState extends SceneObjectState {
   inspectPanelKey?: string;
   /** Panel to view in full screen */
   viewPanelKey?: string;
-  /** Scene object that handles the current drawer */
+  /** Scene object that handles the current drawer or modal */
   drawer?: SceneObject;
```

Comment:
Nit
```suggestion
  overlay?: SceneObject;
```

Reply:
@dprokop yea, agree with this was thinking about renaming it but forget about it :)